spitexorganizer/src/main/java/ch/bfh/bti7081/s2019/black/spitexorganizer/appointment/view/dtos/AppointmentDto.java

```diff
@@ -14,6 +15,11 @@
     private List<TaskDto> tasks;
     private PatientDto patient;
     private ReportDto report;
+    static final private DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm");
+    
+    public String getName() {
```

Comment:
getName of what? the appointment doesn't have a name

Reply:
Ja nicht getter für subklassen machen!
Du kannst es so lösen mit lmbdas:

grid.addColumn(appointmentDto -> appointmentDto.getPatient().getName() + " "  + appointmentDto.getPatient().getSurname()).setHeader("NAME").setWidth("200px");
        grid.addColumn(appointmentDto -> appointmentDto.getDate()).setHeader("Datum");
        grid.addColumn(appointmentDto -> appointmentDto.getStartTime()).setHeader("Von");
        grid.addColumn(appointmentDto -> appointmentDto.getEndTime()).setHeader("Bis");

public String getStartTime(){
        return start.format(DateTimeFormatter.ofPattern("HH:mm"));
    }

    public String getEndTime(){
        return end.format(DateTimeFormatter.ofPattern("HH:mm"));
    }

    public String getDate(){
        return start.format(DateTimeFormatter.ofPattern("dd.MM.yyyy"));
    }